app/forms/models/forms.py

```diff
@@ -246,9 +247,28 @@ class Submission(BaseModel, BasePermissionModel):
     def __str__(self):
         return f"{self.user.user_id}'s submission to {self.form}"
 
+    def send_email(self):
+        from app.communication.notifier import send_html_email
+        from app.util.mail_creator import MailCreator
+
+        send_html_email(
+            self.email_receiver_on_submit,
```

Comment:
Denne må vel være en liste
```suggestion
            [self.email_receiver_on_submit],
```